docs/architecture/lb/nginx.md

```diff
@@ -0,0 +1,270 @@
+# NGINX
+
+NGINX is a popular solution, but there are important disadvantages to consider when using it for scaling syslog ingestion:
+
+- **Uneven TCP traffic distribution**: Even with round-robin load balancing, TCP traffic may not be evenly distributed, leading to overloaded instances. This can cause growing queues, delays, data loss, and potential memory or disk issues.
+  
+- **UDP limitations**: As UDP is a lossy protocol, it’s recommended to send data directly from the source to a nearby syslog server. Using a load balancer can introduce another point of data loss.
```

Comment:
**UDP limitations**: UDP is a protocol prone to data loss. Load balancers can introduce another point of data loss. To mitigate this, send data directly from the source to a nearby syslog server.